src/Administration/Login/Config/LoginConfig.php

```diff
@@ -15,13 +15,17 @@ final class LoginConfig
      * @param non-empty-string $clientSecret
      * @param non-empty-string $redirectUri
      * @param non-empty-string $baseUrl
+     * @param non-empty-string $authorizeEndpoint
+     * @param non-empty-string $tokenEndpoint
      */
     public function __construct(
         public readonly bool $useDefault,
         public readonly string $clientId,
         public readonly string $clientSecret,
         public readonly string $redirectUri,
         public readonly string $baseUrl,
+        public readonly string $authorizeEndpoint,
+        public readonly string $tokenEndpoint,
```

Comment:
Are these full URLs including the domain? Are they relative? Do they include the slash at the start?

Suggestion: Rename these to something like `$authorizeRelativeURL` and during "parsing" (validate) make sure they start with a `/`

Reply:
i renamed it to *_path